spec/models/workshop_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Workshop, type: :model do
```

Comment:
I see you're not using the factory (which if you do it on purpose is fine by me). But then, please remove the factory for it :)